tools/wake/main.cpp

```diff
@@ -316,8 +316,7 @@ int main(int argc, char **argv) {
   std::ofstream log_file("wake.log", std::ios::app);
   auto log_file_defer = wcl::make_defer([&log_file]() { log_file.close(); });
   wcl::log::subscribe(std::make_unique<wcl::log::FormatSubscriber>(log_file.rdbuf()));
-  wcl::log::subscribe(std::make_unique<wcl::log::FatalEventSubscriber>(std::cerr.rdbuf()));
-  wcl::log::info("Initialized logging");
+  wcl::log::info("Initialized logging")();
```

Comment:
Don't we still need a subscriber to display urgent messages?

Reply:
Yeah, I want to do that as part of #1286 since it does a slightly different version of this already. 

Mainly that's just to minimize the merge conflicts, and make this PR a bit smaller